### DISCUSSION TITLE
README.md如何挂载根目录?

### DISCUSSION THREAD

**Discussion by Goojoe** [AlistGo/alist#1975] [Q&A] [answered]
添加网盘挂载都是文件夹,demo站点https://pan.nn.ci/ 是怎么挂载根目录README的?

**Comment by 631894570** [comment]
不清楚作者怎么做的，我在元信息里面添加一个根目录的项，说明里添加markdown

**Reply by shmilyabyy** [reply]
老哥我添加了为什么显示不出来

**Comment by anwen-anyi** [comment, marked as answer]
去元信息
路径写 /
说明哪里 写README内容

**Reply by igbh** [reply]
那里是直接写markdown语句，我想知道能不能在alist的首页里直接加载一个 md文件？就像放在alist某目录下的readme.md那样。

**Reply by anwen-anyi** [reply]
> 那里是直接写markdown语句，我想知道能不能在alist的首页里直接加载一个 md文件？就像放在alist某目录下的readme.md那样。

直接放目录不行，（因为你根目录是好多存储文件一起的 你没法放），但是你可以在元信息放一个 .md的文件链接就是了  路径选择root

**Reply by igbh** [reply]
> > 那里是直接写markdown语句，我想知道能不能在alist的首页里直接加载一个 md文件？就像放在alist某目录下的readme.md那样。
> 
> 直接放目录不行，（因为你根目录是好多存储文件一起的 你没法放），但是你可以在元信息放一个 .md的文件链接就是了 路径选择root

能不能帮忙给个示例？？这个文件链接是怎么写的？就是markdown的文件链接？那在首页显示的是链接还是md文件的内容？？

**Comment by igbh** [comment]
好的，找到方法了，就是放一个形如：http://192.168.2.7:5244/d/%E6%9C%AC%E6%9C%BA%E5%AD%98%E5%82%A8/.md?sign=2IexCybfRrm6i6lv9q5ze33qjjOjmoBzwlPJloY9f78=:0
的链接就可以了，这个md文件就是放在alist某目录里的
